### PR TITLE
Yelp demo and update to summary parameters

### PR DESCRIPTION
**Pull request by ofermend**
added yelp config
added support for 
* customizable summary parameters: numresults and numsentences 
* Moved default_language also under summary config

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,6 +171,22 @@ enable_source_filters: True
 sources: "BBC,NPR,FOX,CNBC,CNN"
 ```
 
+#### summary paramaters (optional)
```
Comment (cjcenizal) [inline README.md:174]: Nit: Capitalize "Summary"

File: README.md [hunk 2]
```diff
@@ -171,6 +171,22 @@ enable_source_filters: True
...
+
+The way summarization works can be configured as follows:
+
+```yaml
+# Default language for summary response (if not specified defaults to "auto)
+summary_default_language: "en"
+
+# number of sentences before and after relevant text segment used for summarization
```
Comment (cjcenizal) [inline README.md:182]: Nit: Capitalize "Number"

File: README.md [hunk 3]
```diff
@@ -171,6 +171,22 @@ enable_source_filters: True
...
+summary_num_sentences: 3
+
+# number of results used for summarization
```
Comment (cjcenizal) [inline README.md:185]: Nit: Capitalize "Number"

File: src/contexts/sendSearchRequest.ts [hunk 4]
```diff
@@ -46,8 +50,8 @@ export const sendSearchRequest = async ({
         numResults: 10,
         corpusKey: corpusKeyList,
         context_config: {
-          sentences_before: includeSummary ? 3 : 2,
-          sentences_after: includeSummary ? 3 : 2,
+          sentences_before: includeSummary ? summaryNumSentences : 2,
+          sentences_after: includeSummary ? summaryNumSentences : 2,
```
Comment (cjcenizal) [inline src/contexts/sendSearchRequest.ts:54]: Oh, interesting... so whatever value is specified for `summaryNumSentences` gets multiplied by 2. I just saw that we do clarify this in the README but I wonder if we can find a name that implies it? `summaryNumSentencesAround`? `summaryNumSentencesBeforeAndAfter`? I dunno, those sound weird to me, too. If we can't think of anything better then let's keep it as-is.

**Review by cjcenizal — APPROVED** [review]
Code LGTM! Didn't test locally. Had a couple nitpicky comments.

File: src/contexts/sendSearchRequest.ts (hunk 4 above)
Comment (ofermend) [reply src/contexts/sendSearchRequest.ts:54]: yes, I agree those are weird. Will leave as such for now and if we find a good name later we can update.

**Pull request merged**